pkg/network/pod/annotations/generator_test.go

```diff
@@ -247,6 +250,166 @@ var _ = Describe("Annotations Generator", func() {
 			Expect(annotations).To(BeEmpty())
 		})
 	})
+
+	Context("Network Info annotation", func() {
+		const (
+			networkName1                     = "boo"
+			networkAttachmentDefinitionName1 = "default/no-device-info"
+			networkName2                     = "foo"
+			networkAttachmentDefinitionName2 = "default/with-device-info"
+			networkName3                     = "doo"
+			networkAttachmentDefinitionName3 = "default/sriov"
+			networkName4                     = "goo"
+			networkAttachmentDefinitionName4 = "default/br-net"
+
+			deviceInfoPlugin    = "deviceinfo"
+			nonDeviceInfoPlugin = "non_deviceinfo"
+		)
+
+		const (
+			multusNetworkStatusWithPrimaryNet = `[{"name":"k8s-pod-network","ips":["10.244.196.146","fd10:244::c491"],"default":true,"dns":{}}]`
+
+			multusNetworkStatusWithPrimaryAndSecondaryNetsWithoutDeviceInfo = `[{"name":"k8s-pod-network","ips":["10.244.196.146","fd10:244::c491"],"default":true,"dns":{}},
+{"name":"default/no-device-info","interface":"pod6446d58d6df","mac":"8a:37:d9:e7:0f:18","dns":{}}]`
+
+			multusNetworkStatusWithPrimaryAndSecondaryNetsWithDeviceInfo = `[{"name":"k8s-pod-network","ips":["10.244.196.146","fd10:244::c491"],"default":true,"dns":{}},
+{"name":"default/with-device-info","interface":"pod2c26b46b68f","dns":{},"device-info":{"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:65:00.2"}}}]`
+
+			multusNetworkStatusWithPrimaryAndSRIOVSecondaryNet = `[{"name":"k8s-pod-network","ips":["10.244.196.146","fd10:244::c491"],"default":true,"dns":{}},
+{"name":"default/sriov","interface":"pod778c553efa0","dns":{},"device-info":{"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:65:00.3"}}}]`
+
+			multusNetworkStatusWithPrimaryAndFourSecondaryNets = `[{"name":"k8s-pod-network","ips":["10.244.196.146","fd10:244::c491"],"default":true,"dns":{}},
+{"name":"default/with-device-info","interface":"pod2c26b46b68f","dns":{},"device-info":{"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:65:00.2"}}},
+{"name":"default/sriov","interface":"pod778c553efa0","dns":{},"device-info":{"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:65:00.3"}}},
+{"name":"default/no-device-info","interface":"pod6446d58d6df","mac":"8a:37:d9:e7:0f:18","dns":{}},
+{"name":"default/br-net","interface":"podeeea394806a","mac":"6a:1f:28:23:58:40","dns":{}}]`
+		)
```

Comment:
PLease move the consts closer to the usage, it is easier to read the test that way.
I think there is only constant that is used twice, I would just define it in each location.

Reply:
Done.